artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AmqpCoreConverter.java

```diff
@@ -283,6 +284,8 @@ protected static ServerJMSMessage processMessageAnnotations(ServerJMSMessage jms
                if (delay > 0) {
                   jms.setLongProperty(HDR_SCHEDULED_DELIVERY_TIME.toString(), System.currentTimeMillis() + delay);
                }
+            } else if (AMQPMessageSupport.HDR_INGRESS_TIME.equals(key) && entry.getValue() != null) {
+               jms.setLongProperty(HDR_INGRESS_TIME.toString(), ((Number) entry.getValue()).longValue());
```

Comment:
Might be good to rename AMQPMessageSupport.HDR_INGRESS_TIME to avoid the qualification and confusing it can cause, and be more consistent with the symbol name created from it?

Reply:
Fixed